src/app/pages/home/home.component.ts

```diff
@@ -12,65 +16,91 @@ import { ICourses } from '../courses';
   providers: [
     Title
   ],
   // Our list of styles in our component. We may add more to compose many styles together
-  styleUrls: [ './home.component.css' ],
+  styleUrls: [ './home.component.scss' ],
   // Every Angular template is first compiled by the browser before Angular runs it's compiler
   templateUrl: './home.component.html'
 })
 export class HomeComponent implements OnInit {
+
   // Set our default values
   public localState = { value: '' };
+  public coursesList;
   // TypeScript public modifiers
   constructor(
     public appState: AppState,
     public title: Title
-  ) {}
-
-  private coursesList = [{
-    id: 1,
-    name: 'kurs 1',
-    time: '1h',
-    date: '16.03.2017',
-    description: 'jakis opis'
-  },{
-    id: 2,
-    name: 'kurs 2',
-    time: '1h',
-    date: '16.03.2017',
-    description: 'jakis opis'
-  },{
-    id: 3,
-    name: 'kurs 3',
-    time: '1h',
-    date: '16.03.2017',
-    description: 'jakis opis'
-  },{
-    id: 4,
-    name: 'kurs 4',
-    time: '1h',
-    date: '16.03.2017',
-    description: 'jakis opis'
-  },{
-    id: 5,
-    name: 'kurs 5',
-    time: '1h',
-    date: '16.03.2017',
-    description: 'jakis opis'
-  },]
+  ) {
+    this.coursesList = [];
+  }
 
   public ngOnInit() {
+    this.coursesList = [{
+      id: 1,
+      name: 'kurs 1',
+      time: '1h',
+      date: '16.03.2017',
+      description: `jakis opis At vero eos et accusamus et iusto odio
+      dignissimos ducimus qui blanditiis praesentium voluptatum deleniti
+      atque corrupti quos dolores et quas molestias excepturi sint occaecati
+      cupiditate non provident, similique sunt in culpa qui officia deserunt
+       mollitia animi, id est laborum et dolorum fuga`
+    }, {
+      id: 2,
+      name: 'kurs 2',
+      time: '1h',
+      date: '16.03.2017',
+      description: `jakis At vero eos et
+       accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium
+       voluptatum deleniti atque corrupti quos dolores et quas molestias
+        excepturi sint occaecati cupiditate non provident, similique sunt in
+        culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga`
+    }, {
+      id: 3,
+      name: 'kurs 3',
+      time: '1h',
+      date: '16.03.2017',
+      description: 'jakis opis'
+    }, {
+      id: 4,
+      name: 'kurs 4',
+      time: '1h',
+      date: '16.03.2017',
+      description: 'jakis opis data'
+    }, {
+      id: 5,
+      name: 'kurs 5',
+      time: '1h',
+      date: '16.03.2017',
+      description: `jakis opis jakis opis jakis opisjakis opis
+      jakis opis jakis opis jakis opis jakis opis jakis opisjakis
+       opisjakis opisjakis opisjakis opisjakis opis`
+    } ];
     console.log('hello `Home` component');
     // this.title.getData().subscribe(data => this.data = data);
   }
 
+  private logData(msg: string) {
+    console.log(msg);
+  }
+
   public submitState(value: string) {
```

Comment:
Zwracaj uwagi na warningi z tslinta, wprawdzie nie psuje to funkcjonalności, ale sprawia, że kod jest bardziej czytelny

`WARNING in ./src/app/pages/home/home.component.ts
[92, 3]: Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.
[98, 3]: Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.
[102, 3]: Declaration of public instance method not allowed after declaration of private instance method. Instead, this should come after constructors.
[8, 8]: You need to leave whitespaces inside of the import statement's curly braces
`